credentials/go10.go

```diff
@@ -0,0 +1,69 @@
+// +build go1.10
```

Comment:
File should be named `go110.go` (and test filename updated to match).

FWIW, I think it's safe to remove go1.9 from travis now, too, and officially call it unsupported.  But we'll do that another time.